test/contract_spec.js

```diff
@@ -0,0 +1,43 @@
+// /*global contract, config, it, assert*/
+/*
+const SimpleStorage = require('Embark/contracts/SimpleStorage');
+
+let accounts;
+
+// For documentation please see https://embark.status.im/docs/contracts_testing.html
+config({
+  //deployment: {
+  //  accounts: [
+  //    // you can configure custom accounts with a custom balance
+  //    // see https://embark.status.im/docs/contracts_testing.html#Configuring-accounts
+  //  ]
+  //},
+  contracts: {
+    "SimpleStorage": {
+      args: [100]
+    }
+  }
+}, (_err, web3_accounts) => {
+  accounts = web3_accounts
+});
+
+contract("SimpleStorage", function () {
+  this.timeout(0);
+
+  it("should set constructor value", async function () {
+    let result = await SimpleStorage.methods.storedData().call();
+    assert.strictEqual(parseInt(result, 10), 100);
+  });
+
+  it("set storage value", async function () {
+    await SimpleStorage.methods.set(150).send();
+    let result = await SimpleStorage.methods.get().call();
+    assert.strictEqual(parseInt(result, 10), 150);
+  });
+
+  it("should have account with balance", async function() {
+    let balance = await web3.eth.getBalance(accounts[0]);
+    assert.ok(parseInt(balance, 10) > 0);
+  });
+}
+*/
```

Comment:
Yeah, we comment the tests out in the templates so that users have a starting point when writing a new test.  Since there is no contract, that's why we comment it.

Reply:
I've implemented it by starting from `embark new MyApp`, like was requested in issue. These test apparently were already commented there for some reason. What I did is: created and ejected new Embark app and rewrote JavaScript frontend in Elm. All other files are left exactly as is.

This file contains tests for Solidity contract, not for JavaScript interface (or Elm in this case), so it's not relevant to this implementation.

TL;DR: It was like that, I didn't touch that :)

Reply:
Thanks for clarification you two!

Ignore my comments. Dismissing my review.